x-pack/plugins/apm/ftr_e2e/cypress/integration/read_only_user/service_overview/errors_table.spec.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import url from 'url';
+import { synthtrace } from '../../../../synthtrace';
+import { opbeans } from '../../../fixtures/synthtrace/opbeans';
+
+const start = '2021-10-10T00:00:00.000Z';
+const end = '2021-10-10T00:15:00.000Z';
+
+const serviceOverviewHref = url.format({
+  pathname: '/app/apm/services/opbeans-java/overview',
+  query: { rangeFrom: start, rangeTo: end },
+});
+
+describe('Errors table', () => {
+  before(async () => {
+    await synthtrace.index(
+      opbeans({
+        from: new Date(start).getTime(),
+        to: new Date(end).getTime(),
+      })
+    );
+  });
+
+  after(async () => {
+    await synthtrace.clean();
+  });
+
+  beforeEach(() => {
+    cy.loginAsReadOnlyUser();
+  });
+
+  describe('when navigating to the service overview', () => {
+    it('errors table is populated', () => {
+      cy.visit(serviceOverviewHref);
```

Comment:
Do you think it makes sense to have one cy.visit call to run all the related tests if they don't change the DOM? I guess this might be a broader question as I have seen that we have multiple cy.visit calls in other tests as well that load the same url. This seems to result in more time running the tests